extensions/wikia/AdEngine/js/config/mobile.js

```diff
@@ -2,17 +2,27 @@
 define('ext.wikia.adEngine.config.mobile', [
 	'ext.wikia.adEngine.adContext',
 	'ext.wikia.adEngine.provider.directGptMobile',
+	'ext.wikia.adEngine.provider.evolve2',
 	'ext.wikia.adEngine.provider.openX',
 	'ext.wikia.adEngine.provider.paidAssetDrop',
 	'ext.wikia.adEngine.provider.remnantGptMobile',
 	require.optional('wikia.instantGlobals')
-], function (adContext, directGptMobile, openX, paidAssetDrop, remnantGptMobile, instantGlobals) {
+], function (
+	adContext,
+	directGptMobile,
+	evolve2,
+	openX,
+	paidAssetDrop,
+	remnantGptMobile,
+	instantGlobals
+) {
 	'use strict';
 
 	var pageTypesWithAdsOnMobile = {
 			'all_ads': true,
 			'corporate': true
-		};
+		},
+		dartEnabled = !instantGlobals || !instantGlobals.wgSitewideDisableGpt;
```

Comment:
Isn't it confusing that `dartEnabled` depends on `wgSitewideDisableGpt`? How about renaming it to `gptEnabled`?